src/CredentialsLoader.php

```diff
@@ -73,9 +73,14 @@ private static function isOnWindows()
     public static function fromEnv()
     {
         $path = getenv(self::ENV_VAR);
-        if (empty($path)) {
+        if (!$path) {
+            $path = $_ENV[self::ENV_VAR];
```

Comment:
this will throw an Out Of Index error if the env var doesn't exist. This would need to be something like
```suggestion
            $path = $_ENV[self::ENV_VAR] ?? null;
```